org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/engine/AnalysisRequirementNotifications.java

```diff
@@ -42,21 +42,21 @@ public static void resetCachedMessages() {
   public static void notifyOnceForSkippedPlugins(org.sonarsource.sonarlint.core.rpc.protocol.common.Language analyzedSkippedLanguage, SkipReason skipReason, String minVersion,
     @Nullable String currentVersion) {
     var languageWithLabel = Language.valueOf(analyzedSkippedLanguage.name());
-    final var shortMsg = "SonarLint failed to analyze " + languageWithLabel.getLabel() + " code";
+    final var shortMsg = "SonarQube failed to analyze " + languageWithLabel.getLabel() + " code";
     if (skipReason == SkipReason.UNSATISFIED_JRE) {
       var content = String.format(
-        "SonarLint requires Java runtime version %s or later to analyze %s code. "
+        "SonarQube requires Java runtime version %s or later to analyze %s code. "
```

Comment:
Same, whenever there is enough place in a sentence, I'd put `SonarQube for Eclipse`. SonarQube should be strictly in locations where it's very hard to fit long words (menu, actions, etc)